src/main/java/com/faforever/client/FafClientApplication.java

```diff
@@ -70,7 +70,7 @@ public static void applicationMain(String[] args) {
     launch(args);
   }
 
-  private static String[] getAdditionalProfiles() {
+    public static String[] getAdditionalProfiles() {
```

Comment:
This should remain private. These profiles are just added to the Spring profiles. So any profile information should be gotten through spring and this should remain private.

Reply:
where is that variable accessed from then?  there is no accessor

Reply:
you launch main, you launch the class in main, and you call this method and the results are now available to any other part of the code and not used

Reply:
i welcome a rewrite to put the pieces in the right places, no ego here.  

the premise was: 
1) if there was a setter for the override, i didn't find it.
2) if the code was running on linux, and the value was blank, it's not going to launch a windows executable anyways, using defaults. 
3) Instead of just checking the library that informs the PROFILE var i made every attempt to give a purpose to the profile var, even if, well, DRY is being violated.